### PR TITLE
Updated chart and tests to allow specify Rancher custom logos persistence

### PR DESCRIPTION
**Pull request by rawmind0**
persistentVolumeClaim or configMap can be used as custom logos persistence volume

https://github.com/rancher/rancher/issues/29419

### CODE REVIEW

**Review by paynejacob — APPROVED** [review]

File: chart/README.md [hunk 1]
```diff
@@ -205,3 +205,9 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
...
+| `customLogos.enabled`          | false                                                 | ***bool*** - Enabled [Rancher custom logos](https://github.com/rancher/ui/tree/master/public/assets/images/logos) pvc. K8s storageClass (dynamic pv) or existingClaim (static pv) are required|
+| `customLogos.volumeKind`       | "persistentVolumeClaim"                               | ***string*** - Use persistentVolumeClaim or configMap for custom logos persitence|
+| `customLogos.volumeName`       | ""                                                    | ***string*** - Use an existing volume. Custom logos should be copied to the volume by the user. Optional for persistentVolumeClaim, required for configMap|
+| `customLogos.storageClass`     | ""                                                    | ***string*** - Set custom logos persistentVolumeClaim storage class|
+| `customLogos.accessMode`       | "ReadWriteOnce"                                       | ***string*** - Set custom persistentVolumeClaim access mode|
+| `customLogos.size`             | "10Gi"                                                | ***string*** - Set custom persistentVolumeClaim size|
```
Comment (paynejacob) [inline chart/README.md:213]: this default value does not match the actual default

**Review by paynejacob — CHANGES_REQUESTED** [review]

File: chart/README.md (hunk 1 above)
Comment (rawmind0) [reply chart/README.md]: Updated

File: chart/README.md [hunk 2]
```diff
@@ -205,3 +205,9 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
...
 | `systemDefaultRegistry`        | ""                                                    | ***string*** - private registry to be used for all system Docker images, e.g., [http://registry.example.com/] *Available as of v2.3.0*                                                                       |
 | `useBundledSystemChart`        | false                                                 | ***bool*** - select to use the system-charts packaged with Rancher server. This option is used for air gapped installations.  *Available as of v2.3.0*                                                       |
+| `customLogos.enabled`          | false                                                 | ***bool*** - Enabled [Rancher custom logos](https://github.com/rancher/ui/tree/master/public/assets/images/logos) pvc. K8s storageClass (dynamic pv) or existingClaim (static pv) are required|
+| `customLogos.volumeKind`       | "persistentVolumeClaim"                               | ***string*** - Use persistentVolumeClaim or configMap for custom logos persitence|
```
Comment (superseb) [inline chart/README.md:209]: typo persitence

Comment (rawmind0) [reply chart/README.md]: Fixed

**Comment by rawmind0** [comment]
Added optional customLogos.privateLabel variable to set [`CATTLE_UI_PL`](https://github.com/rancher/ui/tree/master/public/assets/images/logos#private-labeling) Rancher env variable

**Comment by rawmind0** [comment]
Removed optional customLogos.privateLabel variable to set CATTLE_UI_PL env variable. Rancher chart `extraenv` var can be used to add any needed env variable

**Review by superseb — DISMISSED** [review]

File: chart/tests/pvc_test.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - hasDocuments:
+      count: 0
+    template: pvc.yaml
+- it: should not create custom-logos pvc if not customLogos.enabled
+  set:
+    customLogos.enabled: false
+  asserts:
+  - hasDocuments:
+      count: 0
+    template: pvc.yaml
```
Comment (paynejacob) [inline chart/tests/pvc_test.yaml:36]: nit: newline

**Review by paynejacob — APPROVED** [review]

**Review by paynejacob — APPROVED** [review]

**Pull request merged**